### ISSUE TITLE
Setting an alias for .python-version 

### ISSUE THREAD

**Issue by aiguofer** [pyenv/pyenv#1364]
One of my favorite features of pyenv is the ability to configure multiple versions and/or virtualenvs in my `.python-version` either locally or global. This is really useful for a variety of scenarios.

The only problem is that this doesn't play nice with tools that display your current virtualenv or pyenv version. For example, powerlevel9k and [this module](https://github.com/aiguofer/pyenv.el) I tweaked for Emacs will concat all the lines with ":". This looks horrible when you have multiple versions set up.

It would be awesome if each `.python-version` could have its own alias (and maybe use a `global` alias for the global version). Maybe the first line could be `<name>:alias` and that could be ignored for determining which versions to use but tools could use it to determine the name to display.

Thoughts?

**Comment by aiguofer** [comment]
I took a stab at what this could look like. Looking around, it seems that `pyenv-version-name` was the ideal candidate for this, however it seems that most everything uses this to determine the actual versions. I decided to instead create a separate command that actually shows the display name. This could in turn be used for tools that want to display the version using `pyenv version-display-name`.

Obviously, there should be other commands for setting these aliases, but I thought I'd just present this for now.
